### PR TITLE
Dealer moves

### PR DESCRIPTION
**Pull request by Jedi-Rogue1**

### CODE REVIEW

File: Documents/TMU/Classes/Software Engineering - CS322E.1/GitHub/source/BlackJackDealer.py [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    #while BlackJack.Hand_Value(dealer) < 17:
+      print('Dealer Hits')
+      BlackJackDealer.dealDealerCards(self,deck)        #   Dealer receives a card.
+      print(BlackJackDealer.printDealerCards(self))
+      Sum = BlackJackDealer.sumDealerCards(self)        #   Find the sum of the Dealer's new hand.
+
+      #   Will be using this commanad later.
+      #Sum = BlackJack.Hand_Value(dealer)
+
+    if Sum > 21:
```
Comment (vondojr) [inline Documents/TMU/Classes/Software Engineering - CS322E.1/GitHub/source/BlackJackDealer.py:55]: Not sure this part is necessary? Andrew wrote a win evaluation code that is based on the existing dealer hand. 

It all depends on how we want to set it up though. Any thoughts?

**Review by vondojr — APPROVED** [review]
I don't see why this wouldn't work. Looks pretty clean! 

Only thing we need to think about upcoming is whether we want to do win evaluations all in Andrew's code or have some in yours... 

Currently we have duplicate code. (If the dealer busts, your code will print "Dealer Busts" and Andrews will also print the same.)

Looks good though!

**Comment by mcmillanjy** [comment]
The code itself looks good. However, it looks like you have some things that I don't think you want to merge into dev_1. Also, you changed the test_base, which I don't know if you meant to. Overall, though, it looks like your code works.

**Comment by Jedi-Rogue1** [comment]
> The code itself looks good. However, it looks like you have some things that I don't think you want to merge into dev_1. Also, you changed the test_base, which I don't know if you meant to. Overall, though, it looks like your code works.

I had to change the test_base to run my test with my BlackJack class. My new test_base does affect the original code in test_base though. I don't think it should be that big of a problem, because all I did was linked the import from test_base to my exact file. Should be an easy fix if someone ended up with it, but I am going to change my test_base back to the original just to be sure. As for the new things I am merging into dev_1, I merge my test_base, my code, my test, and the test I had from the previous sprint test_John.py. I forgot to merge test_John.py the last time. I am going to rename the file as well.

**Review by mcmillanjy — APPROVED** [review]

**Pull request merged**